src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -2476,6 +2473,21 @@ - (void)_setupSubscriptionWithReason:(NSString *)reason
                                },
                                ^(void) {
                                    MTR_LOG("%@ got subscription established", self);
+                                   std::lock_guard lock(self->_lock);
+
+                                   // First synchronously change state
+
+                                   // reset subscription attempt wait time when subscription succeeds
```

Comment:
This comment should be next to the `_lastSubscriptionAttemptWait = 0;` line, right?

Reply:
Moved in the last commit